### ISSUE TITLE
[arm64] UISwitchOnOff not lightened by click. 

### ISSUE THREAD

**Issue by p0nce** [AuburnSounds/Dplug#496]
To reproduce: plugin with UISwitchOnOff (eg: signed Panagement) + arm64 + Big Sur + Reaper arm64.
Clicking on reverb or delay on/off successfully set the parameter, but the animation doesn't work correctly.
This seems to be possibly the time flowing too slowly?
Also the PGMT-400 diode don't emit light when clicked.

**Comment by p0nce** [comment]
Likewise, Panagement has no phase display.

**Comment by p0nce** [comment]
Repro script
```sh
#!/bin/sh
dplug-build -a arm64 -c VST-FULL
codesign -s "my identity" builds/macOS-arm64-VST-FULL/Auburn\ Sounds\ Panagement\ 2.vst
lldb /Applications/REAPER.app%       
```

**Comment by p0nce** [comment]
onAnimate not called

**Comment by p0nce** [comment]
Discovery here: https://github.com/ldc-developers/ldc/issues/3603

Should probably review casts to int or uint in Dplug.

**Comment by p0nce** [comment]
fixed in v10.0.1
